test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -1284,9 +1289,9 @@ class test_ucp_sockaddr_wireup_fail_try_next_cm :
         return 0;
     }
 
-    virtual bool wait(entity &e, uint32_t wait_ep_flags)
+    virtual bool test_all_ep_flags(entity &e, uint32_t flags)
```

Comment:
`const` otherwise `virtual` is meaningless

Reply:
it was fixed in the last commit